pkg/controller/plan/kubevirt.go

```diff
@@ -1584,6 +1585,29 @@ func (r *KubeVirt) guestConversionPod(vm *plan.VMStatus, vmVolumes []cnv.Volume,
 	if err != nil {
 		return
 	}
+	if vm.RootDisk != "" {
+		vmInventory := &model.VM{}
+		err = r.Source.Inventory.Find(vmInventory, vm.Ref)
+		if err != nil {
+			err = liberr.Wrap(err, "vm", vm.Ref.String())
+			return
+		}
+
+		var rootDisk string
+		for diskNum, disk := range vmInventory.Disks {
+			if disk.File == vm.RootDisk {
```

Comment:
@liranr23 do you remember if the order of the disk here is consistent? I remember we talked about soothing similar when working on the OVA feature, but the question if we can rely on the disk order inside the VM or we should sort it beforehand?

Reply:
i wouldn't count on it to be consistent. it depends on the inventory which depends on the way we get it from the source provider and that depends on the API itself. there are many places that may change.
however, we do use the VM CR in `podVolumeMounts` function above, there it should be consistent, if you find it helpful.

Reply:
@liranr23 cool thanks, will look into that.

Reply:
> i wouldn't count on it to be consistent.

if the disk order is not constant, a vm may have some disk as 'sda' on one boot, and then it will be 'sdc' on a second boot ? can this work, what am i missing ?

Reply:
@yaacov do you mean on the vm itself? if the vm itself is already posted nothing is gonna change there.

Reply:
> do you mean on the vm itself?

Thanks, I talked with @bkhizgiy and the conclusion was that the UI will set a number "1", "2" and mtv will do the logic to make sure "1" will map to the first disk "2" to the second, etc.